### PR TITLE
Update misc.yaml

### PR DESCRIPTION
**Pull request by hangst**
Frame 4218 belongs to explosion/nuke sequence, which should not part of crate-effects/hide-map sequence.

### CODE REVIEW

File: mods/d2k/sequences/misc.yaml [hunk 1]
```diff
@@ -232,8 +232,8 @@ crate-effects:
 		Start: 4200
 		Length: 18
 	hide-map: DATA.R8
-		Frames: 4218, 4217, 4216, 4215, 4214, 4213, 4212, 4211, 4210, 4209, 4208, 4207, 4206, 4205, 4204, 4203, 4202, 4201
-		Length: 18
+		Frames: 4217, 4216, 4215, 4214, 4213, 4212, 4211, 4210, 4209, 4208, 4207, 4206, 4205, 4204, 4203, 4202, 4201
+		Length: 17
```
Comment (abcdefg30) [inline mods/d2k/sequences/misc.yaml]: It looks to me like frame 4200 is also part of the animation, so I would suggest
```suggestion
		Frames: 4217, 4216, 4215, 4214, 4213, 4212, 4211, 4210, 4209, 4208, 4207, 4206, 4205, 4204, 4203, 4202, 4201, 4200
		Length: 18
```

Comment (hangst) [reply mods/d2k/sequences/misc.yaml]: Yes, I agree

**Review by abcdefg30 — APPROVED** [review]
Lgtm now. (I also took the liberty to squash the commits and give the new commit a better commit title, since you seem to be editing from the GitHub web frontend. Fwiw we usually recommend using a git client locally, but that is more advanced, so dunno if you want to do it.)

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/5e0bd8edb5935c3e7dc144828c3369437d7c0079...928dac3b534c75b6c5645ad8c8b2fe8c8a33fcc2

**Pull request merged**